nft/nfts.yaml

```diff
@@ -995,6 +1029,48 @@ components:
               trait_type:
                 type: string
           description: 'Object - Traits/attributes/characteristics for each NFT asset.'
+        media:
+          $ref: '#/components/schemas/media'
+    ownedContract:
+      type: object
+      properties:
+        address:
+          description: 'Address of the held contract'
+          type: string
+        ownedCount:
```

Comment:
what does this mean exactly? owned count for what? wonder if we should consider a more specific param name?

can we also add a description here